### PR TITLE
♻️ Migrate gallery 35..39 to controls

### PR DESCRIPTION
**Pull request by kvchari**
Partial: https://github.com/ampproject/amphtml/issues/35923

### CODE REVIEW

**Review by alanorozco — APPROVED** [review]
LGTM. Thank you!

File: build-system/test-configs/forbidden-terms.js
Comment (alanorozco) [inline build-system/test-configs/forbidden-terms.js:743]: We should be able to approve this file but I messed up the ownership. Please pull after merging #37751, and we should be able to merge this PR.

Comment (kvchari) [reply build-system/test-configs/forbidden-terms.js:743]: I rebased and force pushed, but I'm still seeing a "missing owners approval" red flag

**Review by rsimha — APPROVED** [review]

**Pull request merged**
